src/ucs/config/parser.h

```diff
@@ -127,6 +127,20 @@ typedef struct ucs_config_bw_spec {
     }; \
     UCS_CONFIG_REGISTER_TABLE_ENTRY(&_table##_config_entry, _list);
 
+#define UCS_CONFIG_DECLARE_TABLE(_table, _name, _prefix, _type) \
+    static ucs_config_global_list_entry_t _table##_config_entry = { \
+        .table  = _table, \
+        .name   = _name, \
+        .prefix = _prefix, \
+        .size   = sizeof(_type) \
+    };
+
+#define UCS_CONFIG_ADD_TABLE(_table, _list) \
+    ucs_list_add_tail(_list, &(_table##_config_entry).list)
```

Comment:
seems some duplication of UCS_CONFIG_REGISTER_TABLE_ENTRY

Reply:
yes for now, macro UCS_CONFIG_REGISTER_TABLE will be removed later when we complete refactoring of initialisation